arches/app/views/search.py

```diff
@@ -180,6 +181,11 @@ def search_terms(request):
 
         if user_is_reviewer is False and index == "terms":
             boolquery.filter(Terms(field="provisional", terms=["false"]))
+            boolquery.filter(
+                Terms(
+                    field="nodegroup_id", terms=[str(ng) for ng in permitted_nodegroups]
+                )
+            )
```

Comment:
This only gets applied if a user isn't a reviewer, but it should probably be applied either way. Otherwise users that are reviewers will get to see terms from restricted tiles.

Reply:
fixed in latest